README.md

```diff
@@ -12,11 +12,11 @@ Follows the [Cordova Plugin spec](https://cordova.apache.org/docs/en/latest/plug
 
 This requires phonegap 7.1.0+ ( current stable v8.0.0 )
 
-    phonegap plugin add phonegap-plugin-barcodescanner
+    phonegap plugin add @transformativeMed/phonegap-plugin-barcodescanner
```

Comment:
Optional, but you could also update the build status link up above -- it's still linked to the original `phonegap` project.

Reply:
sure @ElliotNB , I missed that line